java/ql/lib/ext/sun.net.www.protocol.http.model.yml

```diff
@@ -0,0 +1,8 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["sun.net.www.protocol.http", "DigestAuthentication", False, "computeDigest", "", "", "Argument[2]", "credentials-password", "hq-generated"]
+      - ["sun.net.www.protocol.http", "DigestAuthentication", False, "encode", "", "", "Argument[1]", "credentials-password", "hq-generated"]
+      - ["sun.net.www.protocol.http", "DigestAuthentication", False, "computeDigest", "", "", "Argument[1]", "credentials-username", "hq-generated"]
```

Comment:
FYI, these still have an empty signature string if you want to change them (I wasn't sure if you missed these on commit https://github.com/github/codeql/pull/13978/commits/d923784ae4f0f401b3d5346ebfb38050b8732b85 or if you're intentionally leaving them as `""`.)

Reply:
Since those methods are unlikely to be modified and only have one implementation, I think they're fine with `""`.